src/Lumi/Components/EditableTable.purs

```diff
@@ -87,8 +87,14 @@ editableTable :: forall row. EditableTableProps row -> JSX
 editableTable = makeStateless component render
   where
     render props =
-      container
-        [ header props.columns
+      let
+        lengthRows = case _ of
+          Left arr -> Array.length arr
+          Right arr -> NonEmptyArray.length arr
+      in container
+        [ if props.readonly && lengthRows props.rows == 0
```

Comment:
`lengthRows` could also be `either Array.length NonEmptyArray.length`, but the `case` works too